tests/data/test941

```diff
@@ -32,6 +32,657 @@ converted
 with
 the
 --crlf option
+0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789
```

Comment:
Maybe instead use a `%repeat[]%` instruction here?

Reply:
Yeah, I was looking for something like that. Will fix.

Reply:
That was a little bit more work than expected. Had to enhance the test runner.